### ISSUE TITLE
Crash: Deleting Bullet List Item

### ISSUE THREAD

**Issue by theck13** [wordpress-mobile/WordPress-Aztec-Android#63]
### Expected
Tapping and holding the backspace key removes the list item.

### Observed
Deleting a bullet list item causes the app to crash.  See the stack trace below for details.

```
08-29 08:02:14.409 20057-20057/org.wordpress.aztec E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.wordpress.aztec, PID: 20057
    Theme: themes:{}
    java.lang.IndexOutOfBoundsException: setSpan (36 ... 44) ends beyond length 40
        at android.text.SpannableStringBuilder.checkRange(SpannableStringBuilder.java:1090)
        at android.text.SpannableStringBuilder.setSpan(SpannableStringBuilder.java:665)
        at android.text.SpannableStringBuilder.setSpan(SpannableStringBuilder.java:658)
        at org.wordpress.aztec.AztecText.handleLists(AztecText.kt:1118)
        at org.wordpress.aztec.AztecText.afterTextChanged(AztecText.kt:1057)
        at android.widget.TextView.sendAfterTextChanged(TextView.java:8004)
        at android.widget.TextView$ChangeWatcher.afterTextChanged(TextView.java:10165)
        at android.text.SpannableStringBuilder.sendAfterTextChanged(SpannableStringBuilder.java:1043)
        at android.text.SpannableStringBuilder.replace(SpannableStringBuilder.java:560)
        at android.text.SpannableStringBuilder.replace(SpannableStringBuilder.java:492)
        at android.text.SpannableStringBuilder.replace(SpannableStringBuilder.java:491)
        at android.view.inputmethod.BaseInputConnection.replaceText(BaseInputConnection.java:685)
        at android.view.inputmethod.BaseInputConnection.setComposingText(BaseInputConnection.java:445)
        at com.android.internal.view.IInputConnectionWrapper.executeMessage(IInputConnectionWrapper.java:340)
        at com.android.internal.view.IInputConnectionWrapper$MyHandler.handleMessage(IInputConnectionWrapper.java:78)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:148)
        at android.app.ActivityThread.main(ActivityThread.java:5461)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
```

### Reproduced
1. Place cursor at end of text.
2. Press and hold backspace key.
3. Notice app crash on bullet list deletion.

#### Tested
LG G4 (LG-H815) on Android 6.0.1 with AztecDemo 1.0

**Comment by khaykov** [comment]
Thanks for the report! Can you share the html you used to get this exception?

**Comment by theck13** [comment]
I just used the HTML in the demo app.  Basically, I just opened the app then pressed and held the backspace key.

**Comment by khaykov** [comment]
Fixed at #71
